StitchCoreServicesTwilio/Tests/StitchCoreServicesTwilioTests/StitchCoreServicesTwilioTests.swift

```diff
@@ -1,6 +1,99 @@
 import XCTest
+import MockUtils
+import MongoSwift
+import StitchCore
 @testable import StitchCoreServicesTwilio
 
+final class MockCoreStitchService: CoreStitchService {
```

Comment:
I don't like how this is embedded directly in the test, or mixed into the test sources of StitchCoreTests. When I have more time (possibly after I'm done looking at Jason and Eric's PRs) I'll create a new target and product from the StitchCore package called StitchCoreMocks that will contain all of these in a way that can be imported without it being part of StitchCore or StitchCoreTests.